### PR TITLE
perf: 开始唤醒增加资源更新Ocr

### LINKED ISSUE #10643: 开始唤醒添加更新 loading 检测
> [!warning]
存在未确认的链接，请谨慎访问。
There are unconfirmed links, please visit with caution.

### 在提问之前...

- [X] 我填写了简短且清晰明确的标题，以便开发者在翻阅 Issue 列表时能快速确定大致问题。而不是“一个建议”、“卡住了”等
- [X] 我基本确定这是一个新功能/建议，而不是遇到了 Bug（不确定的话请附上日志）

### 说说你遇到的问题？

_No response_

### 有什么好的想法？

[MuMu12-20240919-160216.zip](https://github.com/user-attachments/files/17055675/MuMu12-20240919-160216.zip)

还有个正在检查资源完整性

### 其他内容

_No response_

### PR DESCRIPTION
**Pull request by status102**
- close #10643 

cc @Constrat 
`MB` for overseas, not sure if it would run steady. merge it yourself if you like

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/cn-bug-report.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ body:
           1. `debug` 文件夹中的 `asst.log` 和 `gui.log` 日志文件，并说明问题出现的大致时间点
             若 asst.log 文件小于 10 KB，则需同时上传 asst.log.bak 文件
           2. `config` 文件夹中的 `gui.json` 配置文件
-        
+
```
Comment (status102) [inline .github/ISSUE_TEMPLATE/cn-bug-report.yaml:47]:
```suggestion
        
```

File: .github/ISSUE_TEMPLATE/en-bug-report.yaml [hunk 2]
```diff
@@ -58,7 +58,7 @@ body:
           1. The `asst.log` and `gui.log` log file in the `debug` folder, and indicate the approximate point in time when the problem occurred.
             If the asst.log file is less than 10 KB, you also need to upload the asst.log.bak file.
           2. The `gui.json` config file in the `config` folder
-        
+
```
Comment (status102) [inline .github/ISSUE_TEMPLATE/en-bug-report.yaml:61]:
```suggestion
        
```

**Comment by Constrat** [comment]
Wait, what is `MB`.
Yesterday on Daydreamer PR i said this:
```
chore: YostarEN text
I haven't found the "releasing neurotransmitter" string in EN. 
Only:
"Loading network configuration"
"Acquiring update"
"Checking the completeness of the resources"
```

**Comment by status102** [comment]
MB is the update data package size

downloading... (xxx MB/ xxxx MB)

**Comment by Constrat** [comment]
OH LMAO YOU ARE RIGHT.
Completely missed that XD

**Review by Constrat — APPROVED** [review]

**Pull request merged**